ci/test-remote.sh

```diff
@@ -20,7 +20,8 @@ main() {
 
     retry cargo fetch
     cargo build
-    export CROSS="${PROJECT_HOME}/target/debug/cross"
+    CROSS=$(cross_bin "${PROJECT_HOME}" debug)
+    export CROSS
     export CROSS_UTIL="${PROJECT_HOME}/target/debug/cross-util"
```

Comment:
This could use something similar to `cross_bin`. We could likely generalize here.